src/styles/card.scss

```diff
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  @include zero-margins-paddings;
+  font-family: $cardsFontStyle;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: $cardsBorderRadius;
+  border: $cardsBorder;
+  display: flex;
+  flex-direction: column;
+
+  background: $white-color;
+
+  &__image {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__title {
+    @include zero-margins-paddings;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: $letter-spacing;
+    text-align: left;
+
+    color: $main-accent-color;
+  }
+
+  &__product-code {
+    @include zero-margins-paddings;
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: $letter-spacing;
+    text-align: left;
+
+    color: $secondary-color;
+  }
+
+  &__rate {
+    margin-bottom: 24px;
+    @include flex-spacebetween-center;
+  }
+
+  &__reviews {
+    @include zero-margins-paddings;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: $letter-spacing;
+
+    color: $main-accent-color;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    @include flex-spacebetween-center;
```

Comment:
```suggestion
    @include flex-spacebetween-center;
    
    &-text {}
    &-value {}
```

Reply:
put it in '&_price'. Not sure i got it right